cfbs/main.py

```diff
@@ -45,6 +45,11 @@ def main() -> int:
         print("")
         user_error("No command given")
 
+    if args.masterfiles and args.command != "init":
+        user_error(
+            "The option --masterfiles is only for cfbs init, not cfbs %s" % args.command
```

Comment:
I think it would read a bit better with quotes:

```suggestion
            "The option --masterfiles is only for 'cfbs init', not 'cfbs %s'" % args.command
```